drivers/can/can_mcp2515.c

```diff
@@ -826,7 +826,7 @@ static int mcp2515_init(struct device *dev)
 	return ret;
 }
 
-#ifdef CONFIG_CAN_1
+#if DT_HAS_DRV_INST(0)
```

Comment:
shouldn't this be inst(1)?

Reply:
No, the first instance is 0.

Reply:
right; is there a reason for the functions right below to have the _1 suffix?

Reply:
The reason is that the instance is called CAN_1 from the MCU point of view